setup.sh

```diff
@@ -57,7 +57,23 @@ function assertAllPrerequisitePresent
     else
         echo -n "$(determineInstructionInstallationAllPrerequisiteMissing "${allPrerequisiteMissing}")"
 
-        exit 1
+        if [${1} ]; then
+            exit 1
+        else
+            assertAllPrerequisitePresent true
+        fi
+    fi
+}
+
+function assertComposerPrerequisitePresent
+{
+    phpModules=$(php -m | head)
+
+    if [[ ${phpModules} != *"curl"* ]]; then
```

Comment:
There are some extension missing according to: [composer.json](https://github.com/bunq/sdk_php/blob/8c1faefc111d9b970168a1837ca165d854954941/composer.json)

Aslo this seems like code repeation. Please create a new method:

1. isModlueMissing(string[] $allModules, string $moduleToFind): bool
 You can do this via:

```